ui/src/app/machines/views/AddMachine/AddMachineFormFields/AddMachineFormFields.js

```diff
@@ -133,7 +136,7 @@ export const AddMachineFormFields = ({ saved }) => {
             setFieldValue("pxe_mac", formatMacAddress(e.target.value));
           }}
           placeholder="00:00:00:00:00:00"
-          required
+          {...macAddressRequired}
```

Comment:
Rather than destructuring this, it might be simpler to just say:

```
const macAddressRequired = values.power_type !== "ipmi";
```
and in the component:

```
required={macAddressRequired}
```